lib/Utils/Graph/Graph.h

```diff
@@ -64,6 +74,66 @@ class Graph {
     return {};
   }
 
+  FailureOr<std::vector<V>> get_longest_source_to_sink_path() {
```

Comment:
nit - could you please camelcake the new functions to match the general code style?

Reply:
It's interesting that the formatting checks don't complain about this... 

PS: typos + GenAI are a magical combination:
![image](https://github.com/user-attachments/assets/d10ec92a-74c2-4625-8d68-df780422f7bb)